wallmatrix/sources/mbta.py

```diff
@@ -7,31 +8,41 @@
 from wallmatrix.sources import Source
 
 
+class Line(TypedDict):
+    color: str
+    line: str
+    route: str
+    origin: str
+    destination: str
+    direction: int
+    walk_min: int
+
+
 class MBTA(Source):
     SOURCE_NAME = "MBTA North from NEU"
 
-    LINES = [
+    LINES: list[Line] = [
         {
             "color": "ED8B00",
             "line": "line-Orange",
             "route": "Orange",
-            "origin": "place-rugg",  # Ruggles
+            "origin": "place-masta",  # Mass Ave
             "destination": "place-haecl",  # Haymarket
             "direction": 1,  # North
-            "walk-min": 6,
+            "walk_min": 6,
```

Comment:
`walk_min` values need to be updated -- this represents the time it takes to get from our place to the corresponding stop.

Reply:
true!